### PR TITLE
readme typo: swap example [skip ci]

### LINKED ISSUE #4015: readme typo in master and newest branch
## 📚 Documentation

For typos and doc fixes, please go ahead and:

On master and the newest branch, variable loss under training_step is referenced before assignment.

```python
class LitAutoEncoder(pl.LightningModule):

    def __init__(self):
        super().__init__()
        self.encoder = nn.Sequential(nn.Linear(28 * 28, 128), nn.ReLU(), nn.Linear(128, 3))
        self.decoder = nn.Sequential(nn.Linear(3, 128), nn.ReLU(), nn.Linear(128, 28 * 28))
    
    def forward(self, x):
        # in lightning, forward defines the prediction/inference actions
        embedding = self.encoder(x)
        return embedding

    def training_step(self, batch, batch_idx):
        # training_step defined the train loop. It is independent of forward
        x, y = batch
        x = x.view(x.size(0), -1)
        z = self.encoder(x)
        x_hat = self.decoder(z)
  
        # the order of underline two lines are incorrect, variable loss referenced before assignment
        # self.log('train_loss', loss)
        # loss = F.mse_loss(x_hat, x)

        loss = F.mse_loss(x_hat, x)
        self.log('train_loss', loss)
        return loss

    def configure_optimizers(self):
        optimizer = torch.optim.Adam(self.parameters(), lr=1e-3)
        return optimizer
````

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

Fixes #4015

## PR review    
Anyone in the community is free to review the PR once the tests have passed.     
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Review by SkafteNicki — APPROVED** [review]

**Review by rohitgr7 — APPROVED** [review]

**Comment by NickYi1990** [comment]
I think there need more changes, the batch_size is default 1 in the example, for user-friendly set it to 128 would be much better.
